shell.nix

```diff
@@ -1,12 +1,12 @@
 { pkgs ? import <nixpkgs> { } }:
 pkgs.mkShell {
   nativeBuildInputs = with pkgs.buildPackages; [
-    cabal-install
+    (haskell.lib.justStaticExecutables (haskellPackages.cabal-install))
     ghc
     ghcid
     haskell-language-server
-    haskellPackages.fourmolu_0_12_0_0
-    pkgconfig
+    (haskell.lib.dontCheck (haskellPackages.callHackage "fourmolu" "0.12.0.0" {}))
```

Comment:
Fourmolu 0.12 no longer included in `haskellPackages` by default.

Reply:
Fix https://github.com/yvan-sraka/cabal.nix/pull/2 :)